### PR TITLE
try to proactively answer questions about difference between network transfer & IP ranges

### PR DESCRIPTION
**Pull request by sebastian-lerner**
@devinmarieb what do you think? too wordy?

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by devinmarieb** [comment]
@sebastian-lerner I don't think it's too wordy. I did update the linking format on one of the links in there and also made a few small changes if you want to take a look in my commit. I think we are trying to move away from capitalizing things like "Network Transfer" unless we are specifying a particular button/tab in the UI.

**Comment by sebastian-lerner** [comment]
hmmmm i am trying to indicate "Network Transfer & Egress" that you see in the Plan Usage page.  I didn't realize that we show customers in the Projects tab a total of per-project Network Transfer & Egress so we may want to adjust the wording to reflect that instead of directing them to the "Network tab".  Want me to take a stab at re-wording?

**Comment by sebastian-lerner** [comment]
Screen Shot 2022-01-13 at 1 20 09 PM

**Comment by thomasr8** [comment]
@sebastian-lerner I just made a few very minor edits and changes that I noticed. These were very small (a typo and a few spacing fixes), so nothing major.

**Comment by devinmarieb** [comment]
@sebastian-lerner Yeah since there's definitely a difference between overall network transfer (which a user can still see in the app) and the billable network transfer, I think the clarification needs to be made. I didn't read this as speaking about billable usage specifically I guess. I still wouldn't capitalize "network transfer" in this instance, but I would consider adding the word "egress", and I think a distinction should be made that we are only talking about billable network transfer and egress here.
